optimade/filtertransformers/tests/test_django.py

```diff
@@ -11,8 +11,8 @@
 test_data = [
```

Comment:
This is most definitely for another PR, but as a general question (to whomever happens to read this), would it not be beneficial for future parser developers/contributors that we have a central place with a list of queries, wherein one could also add their parser's expected output for testing? E.g. a python dictionary or importable JSON file?

Reply:
That's a good idea.  But rather having a dictionary I would prefer a python template file which contains all the cases. The dictionary/JSON can easily become quite complex when you start to categorise the test cases or you want to test failures with different exceptions etc...

Reply:
related issue(#80)